source/protobuf/data_moniker.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
```

Comment:
The description says:

> We would reference the source files directly from the sideband repo as a subsequent PR and remove thse from the source in the grpc-device repo.

What files specifically will be moved to the sideband repo? This `proto`? Will you also revert the changes to `core_server` and `server_confguration` and move that to the `grpc-sideband` repo?

Reply:
There is more "sideband" code than I expect added in this change. If sideband is included as a submodule. Doesn't that have the proto file and server implementation?

Reply:
Only the proto and the data_moniker service will be referenced directly by the grpc-device code. 

The core-server and server registartion has to live here since we are creating the service inside the grpc-device server.

The sideband repo code serves to 

1. provide utility class for managing the sideband communication.
2. provide moniker service implementation and its interface for clients (.proto file).